src/client/cl_console.c

```diff
@@ -89,6 +89,8 @@ static void Cl_DrawConsole_Buffer(void) {
 		g_free(lines[i]);
 		y += ch;
 	}
+
+	Mem_Free(lines);
 }
```

Comment:
I was worried I missed one of these, I thought they were all buffered. In the case of one called per frame, I'd recommend using alloca instead - it allocates on the stack, and is the closest to what the VLA accomplished. If need be, we can #if _MSC_VER it as well so that it's not required for non-MSVC.